packages/components/src/ConditionalFilter/group-filter.scss

```diff
@@ -1,6 +1,7 @@
 @import '~@redhat-cloud-services/frontend-components-utilities/styles/_all.scss';
 
 button.pf-c-menu-toggle {
```

Comment:
I think this breaks becayse of usage of global styling. Can we scope it with some className? This can break other elements on screen with the same selector.

I believe that was the reason of the broken padding.